src/shared/components/VideoPreview/VideoPreview.tsx

```diff
@@ -35,6 +35,8 @@ type VideoPreviewProps = {
   duration?: number
   // video watch progress in percent (0-100)
   progress?: number
+  removeButton?: boolean
+  handleRemoveInterruptedVideo?: () => void
```

Comment:
Let's make it consistent with the rest of the handlers and name it `onRemoveButtonClick` (we can also pass the event as the rest of the handlers.

This also has a nice benefit that `VideoPreview` doesn't have to know about `interruptedVideo` - it just has a remove button and it can be also reused for other purposes